flo-runner/src/main/java/com/spotify/flo/context/Logging.java

```diff
@@ -28,17 +28,21 @@
 import com.spotify.flo.TaskInfo;
 import com.spotify.flo.freezer.Persisted;
 import com.spotify.flo.status.TaskStatusException;
+import java.io.Serializable;
 import java.time.Duration;
 import java.util.Optional;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 import org.slf4j.Logger;
 
-class Logging {
+class Logging implements Serializable {
```

Comment:
In beam, they solve this problem by using memoizable suppliers (from guava) that are serializable. I would rather refactor the code to use them, then making `Logging` serializable.

Reply:
It was a cheap hack to put stateful task status tracking into the `Logging` class to begin with. Refactoring.